src/Bicep.Core/Syntax/SyntaxFactory.cs

```diff
@@ -245,5 +253,83 @@ private static string EscapeBicepString(string value)
             TokenType.NullKeyword => "null",
             _ => ""
         };
+
+        public static SyntaxBase FlattenStringOperations(SyntaxBase original)
+        {
+            if (original is FunctionCallSyntax functionCallSyntax)
+            {
+                if (functionCallSyntax.NameEquals("concat"))
```

Comment:
>NameEquals [](http://example.com/codeflow?start=39&length=10)

Same name binding comment here and below.

Reply:
https://github.com/Azure/bicep/issues/2773